tests/unit/test_project_structure.py

```diff
@@ -475,6 +475,28 @@ def test_create_validation(self):
                 '''
             )
 
+    def test_describe(self):
+        self.run_sql(
+            '''
+                CREATE PREDICTOR mindsdb.pred
```

Comment:
`CREATE PREDICTOR` is obsolete, or not?

Reply:
updated PREDICTOR -> model in all test file